lib/Target/AVR/AVRTargetMachine.cpp

```diff
@@ -25,7 +25,7 @@
 namespace llvm {
 
 /// Processes a CPU name.
-static StringRef getTargetCPU(StringRef CPU) {
+static StringRef GetTargetCPU(StringRef CPU) {
```

Comment:
LLVM convention is to have this `lowerCase`, I changed this recently due to an upstream code review.

Would you be able to change the case _and_ rename the method? `getTargetCPU` clashes with a function inside the superclass. Maybe `getCPU` or something.